tests/integration/test_hana.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.exc import OperationalError
+
+from datahub_sap_hana.ingestion import HanaConfig, HanaSource
+from datahub.ingestion.api.common import PipelineContext
+
+from datahub.emitter import mce_builder
+from datahub.emitter.mcp_builder import mcps_from_mce
+from tests.test_helpers import mce_helpers
+from tests.test_helpers.type_helpers import PytestConfig
+
+from tests.test_helpers.click_helpers import run_datahub_cmd
+import yaml
+
+
+@pytest.fixture
+@pytest.mark.integration
```

Comment:
This is on the wrong function, you want it on the two functions below that start with `test_`